### PR TITLE
chore: update lint-staged

### PR DESCRIPTION
**Pull request by abdonrd**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -105,12 +105,10 @@
   "lint-staged": {
     "*.js": [
       "eslint --fix",
-      "prettier --write --ignore-path .eslintignore",
-      "git add"
+      "prettier --write --ignore-path .eslintignore"
     ],
     "*.md": [
-      "prettier --write --ignore-path .eslintignore",
-      "git add"
```
Comment (LarsDenBakker) [inline package.json:113]: is git add no longer needed with lint-staged v10? @daKmoR I think you know more about this than me?

Comment (daKmoR) [reply package.json:113]: yes it's no longer needed 🤗

Comment (abdonrd) [reply package.json:113]: Exactly! https://github.com/okonet/lint-staged/releases/tag/v10.0.0

**Comment by daKmoR** [comment]
thank you for this update 🤗

**Pull request merged**